pylint/extensions/check_elif.py

```diff
@@ -12,7 +12,7 @@ class ElseifUsedChecker(BaseTokenChecker):
     """
 
     __implements__ = (ITokenChecker, IAstroidChecker)
-    name = 'elseifused'
+    name = 'else_if_used'
```

Comment:
I can revert this if you'd like - this was just an awkwardness I noticed with the docs generation, because (for builtin and extension checkers alike) the docs use `checker_name.replace("_", " ").title()` so this was giving "Elseifused checker" rather than "Else If Used checker" as the section header. Let me know what you'd prefer.